trust/save.c

```diff
@@ -105,7 +106,9 @@ p11_save_open_file (const char *path,
 	if (asprintf (&temp, "%s%s.XXXXXX", path, extension) < 0)
 		return_val_if_reached (NULL);
 
+	mode = umask (066);
```

Comment:
It would be nice to have the intended permissions documented explicitly as the `066` is not very explanatory. btw. `066` seems wrong as I'd have expected 3 digits after zero. It also ignores the running user umask which may be stricter than what you set there (not know if that could be an issue, but it is not apparent to me how this function is used).

Reply:
I agree that we should document it (as a comment at least). For other two points: the C standard only says that a literal starting with `0` is treated as octal (so 066 == 0066). Also, I thought that a new umask is ORed with the existing umask so it would never become more permissive.

Reply:
Sorry, the last part was my misunderstanding. I guess using `0077` would be better, as it is strictest and self-explanatory.

Reply:
ok